### DISCUSSION TITLE
Plot a graph using sensor attributes for x and y-axis

### DISCUSSION THREAD

**Discussion by jjvelar** [dbuezas/lovelace-plotly-graph-card#465] [Q&A]
Hi,
I have a sensor named sensor.aigues_de_catalunya_lecturas with
state = "true"
attribute "dates" = ["14/01/2020","09/04/2020","14/07/2020","14/10/2020","12/01/2021","13/04/2021","13/07/2021","11/10/2021","12/01/2022","14/04/2022","13/07/2022","13/10/2022","11/01/2023","11/04/2023","13/07/2023","11/10/2023","09/01/2024","09/04/2024","10/07/2024","09/10/2024"]
atribute "values" = [55,65,66,70,43,53,62,55,59,49,55,49,50,44,42,41,43,37,47,41]

How can I create a chart with "dates" values in the x-axis and "values" values in the y-axis?

Thanks in advance for your help

**Comment by dbuezas** [comment]
Hi there 👋 
See here: https://github.com/dbuezas/lovelace-plotly-graph-card/discussions/458#discussioncomment-10888440
